### PR TITLE
Fix/null data

### PR DESCRIPTION
**Pull request by rahulinux**

### CODE REVIEW

**Comment by rahulinux** [comment]
@bartpelgrim  This is for those false alarm about ecflow service we get, I see that error we get because if ecflow has created new cycle and there is no data in it then this tool get failed and unable to send data to cloud watch and then cloud watch complaint about no data point.

File: ecflow_cloudwatch/ecflow_state_parser.py [hunk 1]
```diff
@@ -129,13 +127,19 @@ def parse(self):
...
-                                available_data.append(','.join(map(lambda x: str(x), [min, max, threshold])))
+                                # Use fetch recent only for meter data 
+                                if self.fetch_new and  self.filter_date(task.get_abs_node_path()):
+                                    for meter in task.meters:
+                                        min, max, threshold = meter.min(), meter.max(), meter.value()
+                                        available_data.append(','.join(map(lambda x: str(x), [min, max, threshold])))
+                                else:
+                                    for meter in task.meters:
+                                        min, max, threshold = meter.min(), meter.max(), meter.value()
+                                        available_data.append(','.join(map(lambda x: str(x), [min, max, threshold])))
```
Comment (bartpelgrim) [inline ecflow_cloudwatch/ecflow_state_parser.py]: This is exacly the same as line 132-134?

Comment (rahulinux) [reply ecflow_cloudwatch/ecflow_state_parser.py]: Fixed!

Comment (bartpelgrim) [reply ecflow_cloudwatch/ecflow_state_parser.py]: 137 and 139 are still the same...

Comment (rahulinux) [reply ecflow_cloudwatch/ecflow_state_parser.py]: Yes, due to old data sets I have tests, I have to add this but I removed it now and refactor test to check only output type

**Pull request merged**